.github/workflows/static.yml

```diff
@@ -43,10 +45,21 @@ jobs:
           bundle exec middleman build
       - name: Setup Pages
         uses: actions/configure-pages@37538b280cbfab35da9dcd936a2b95f1f522d927
+      - name: Prepare archive for upload
+        shell: sh
+        run: |
+          chmod -c -R +rX "build"
```

Comment:
Hm what does this line do? There's no "build" directory, and running it locally I get 
```
> chmod -c -R +rX "build"
chmod: illegal option -- c
```

Reply:
This fixes any directory permissions prior to archiving, "build" is generated by middleman earlier in the workflow.

`chmod` might behave differently on your Mac versus the action runner which is Ubuntu.